mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -89,22 +94,21 @@ def _check_predicted_data(
                 "in conformity with the Gamma distribution support."
             )
 
+    @staticmethod
     def _all_strictly_positive(
-        self,
         y: ArrayLike,
     ) -> bool:
-        if np.any(np.less_equal(y, 0)):
-            return False
-        return True
+        return not np.any(np.less_equal(y, 0))
 
     def get_signed_conformity_scores(
         self,
+        X: ArrayLike,
         y: ArrayLike,
         y_pred: ArrayLike,
     ) -> NDArray:
         """
-        Compute samples of the estimation distribution from the predicted
-        values and the conformity scores, from the following formula:
+        Compute the signed conformity scores from the observed values
+        and the estimator, from the following formula:
```

Comment:
UPD: this description must be updated (no `estimator` here).

Reply:
Done